.circleci/setup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Install PHP extensions
+sudo docker-php-ext-install pdo_mysql
+
+# Install extension
+sudo apt-get install -y libpng-dev
+
+# Install PHP Extensions
+sudo docker-php-ext-install gd
+
+# Install Composer
+'curl -sS https://getcomposer.org/installer | sudo php -- --install-dir=/usr/local/bin --filename=composer'
+
+# Display versions
+php -v
+composer --version
+
+# Install mysql-client
+sudo apt-get install mysql-client
+
+# Configure bash environment variables
+echo 'export PATH=~/.composer/vendor/bin:~/drush:$PATH' >> $BASH_ENV
+echo 'export HOME=/tmp/drush-sandbox/home' >> $BASH_ENV
```

Comment:
Hmm. My understanding is that all we need to do is assure that TEMP env variable is available. Unish does the rest.

Reply:
Yes, I should have commented on this. The problem is that on Circle CI 2.0 with composer-highest.json, when Unish launches `drush` in a subprocess, somehow the `HOME` environment variable is whatever it was set to by the process that called Unish, regardless of what Unish tries to set it to. It is very odd that a) this does not seem to happen with the composer.json test (implying that the cause is in symfony/process), although b) this does not seem to happen with any test in the old Travis CI scripts. Setting HOME to what the drush subprocess expects it to be is a workaround, but this would break again if the Unish path calculation for `HOME` changed again in the future. 😞

Reply:
I guess this is related to the comment in CommandUnishTestCase::execute()